src/django/api/models.py

```diff
@@ -2490,6 +2492,36 @@ def post_save(sender, **kwargs):
             index_extendedfields([instance.facility.id])
 
 
+class Event(models.Model):
+    EVENT_TYPE_CHOICES = (
+        ('CREATE', 'CREATE'),
+        ('UPDATE', 'UPDATE'),
+        ('DELETE', 'DELETE'),
+        ('MERGE', 'MERGE'),
+        ('SPLIT', 'SPLIT'),
+        ('MOVE', 'MOVE'),
+        ('OTHER', 'OTHER'),
+        ('ASSOCIATE', 'ASSOCIATE'),
+        ('DISSOCIATE', 'DISSOCIATE'),
+        ('CLAIM', 'CLAIM'),
+        ('CLAIM_UPDATE', 'CLAIM_UPDATE'),
+        ('CLAIM_REVOKE', 'CLAIM_REVOKE'),
```

Comment:
Is it just coincidence that these match this constant?
https://github.com/open-apparel-registry/open-apparel-registry/blob/7833460527f082bfe8c77e85582347c03a368e7e/src/django/api/constants.py#L73-L85

If they're unrelated but just have similar choices the way things are is fine, but if they're related to each other (i.e. these are events for those actions) then I'd suggest directly using the same constants here:

```python
    EVENT_TYPE_CHOICES = (
        (FacilityHistoryActions.CREATE, FacilityHistoryActions.CREATE)
        ...,
    )
```

Reply:
The Event model is intended to eventually replace the Facility History API, so it is using the same actions. I've switched to using the constants.